composeApp/build.gradle.kts

```diff
@@ -11,6 +11,30 @@ plugins {
     alias(libs.plugins.ktlint)
 }
 
+val flavor: String by project
```

Comment:
Should it be `flavor` or something like `organization`? Cause we have other flavors too.